HousingSearchApi/V1/Boundary/Responses/Transactions/GetTransactionListResponse.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using Hackney.Shared.HousingSearch.Domain.Transactions;
+
+namespace HousingSearchApi.V1.Boundary.Responses.Transactions
+{
+    public class GetTransactionListResponse
+    {
+        private long _total;
+
+        public List<Transaction> Transactions { get; set; }
```

Comment:
I would make this private and use the static factory method approach, and manipulate the list (by adding more items) through a method within this class.